src/Protocol/Protocol.h

```diff
@@ -15,6 +15,7 @@
 #include "../Map.h"
 #include "../ByteBuffer.h"
 #include "../EffectID.h"
+#include "../ClientHandle.h"
```

Comment:
It's needed to complete the type for accessing m_Client in cProtocol_1_10_0::HandlePacketStatusRequest:

```c++
	m_Client->ForgeAugmentServerListPing(ResponseValue); 
```

but, I could move it to Protocol_1_10.cpp closer to where it is needed